tests/bats/20_collections.bats

```diff
@@ -58,15 +58,49 @@ teardown() {
     refute_line "crowdsecurity/mysql"
 }
 
-@test "cannot remove a collection twice" {
-    run -0 cscli collections install crowdsecurity/mysql -o human
-    run -0 --separate-stderr cscli collections remove crowdsecurity/mysql
-    run -1 --separate-stderr cscli collections remove crowdsecurity/mysql -o json
-    run -0 jq -r '.level' <(stderr)
-    assert_output 'fatal'
-    run -0 jq -r '.msg' <(stderr)
-    assert_output --partial "unable to disable crowdsecurity/mysql"
-    assert_output --partial "doesn't exist"
+@test "cannot remove a collection that belongs to another" {
```

Comment:
can you check `--force` works tho ?

Reply:
yes, as I said in the comment below. anything else?